src/dart_diagnostic_provider.ts

```diff
@@ -17,7 +17,7 @@ export class DartDiagnosticProvider {
 
 	private handleErrors(notification: as.AnalysisErrorsNotification) {
 		let errors = notification.errors;
-		if (!getConfig('showTodos'))
+		if (!getConfig<boolean>('showTodos'))
```

Comment:
[Sorted](https://github.com/DanTup/Dart-Code/commit/e06b06ab7df3f817d5a82da6e7da3e3a996e31b4).

I also moved `verbose` to config (hidden). This way you can turn on and off without a recompile, it applies immediately. It's off by default (since hiddens don't have defaults) but that seems reasonable.